### PR TITLE
Bmcdole/validation tooltips

### PR DESCRIPTION
**Pull request by bamcdole**
I've been picking away at these validation TODO's in the last hour of the past three days. This adds validation tooltips to the rest of the wizard fields, and adds a validation step to the Delivery Folder field.

The work is captured in this ticket: US122093

Didn't finish in time to be tested in this sprint, so we can pull it into the next non-inspiration sprint.

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -33,8 +33,11 @@ export default {
 	'step1.title': 'Select Your Advanced Data Set',
 	'step1.scheduleName.label': 'Schedule Name',
 	'step1.scheduleName.placeholder': 'Enter a name',
+	'step1.scheduleName.errorMessage': 'Schedule Name cannot be blank',
+	'step1.scheduleName.tooLong': 'Schedule Name cannot be blank',
```
Comment (JeremyWinkler) [inline lang/en.js:37]: Is this used anywhere?

Comment (bamcdole) [reply lang/en.js:37]: Nope, will remove.

File: src/components/d2l-delivery-method.js [hunk 2]
```diff
@@ -138,19 +168,27 @@ class DeliveryMethod extends LocalizeMixin(LitElement) {
...
+		this._validateFolder();
 		this._commitChanges();
 	}
 
 	_validateDeliveryMethod() {
 		this.invalidDeliveryMethod = this.deliveryMethod === null;
 	}
+
+	_validateFolder() {
+		const validDirectoryTest = /^[a-zA-Z].*/;
```
[the remark this answers is not in the document]
Comment (bamcdole) [reply src/components/d2l-delivery-method.js:187]: Will revisit

**Comment by bamcdole** [comment]
image

**Review by lweber-d2l — DISMISSED** [review]

**Pull request merged**